docs/wiki/Whats-new.md

```diff
@@ -1297,4 +1297,4 @@ Note that a number of initiatives have been updated that will fail to deploy if
 
 #### Other
 
-- Contoso Reference Implementation Update - Virtual WAN Hub default CIDR changed from `/16` to `/23` - closing issue [#440](https://github.com/Azure/Enterprise-Scale/issues/440)
+- Contoso Reference Implementation Update - Virtual WAN Hub default CIDR changed from `/16` to `/23` - closing issue [#440](https://github.com/Azure/Enterprise-Scale/issues/440)
```

Comment:
What's this?

Reply:
WIKI page updated
This line is something I had to align my version with. It is in the official page ... I just don't know why it is appearing this way